API Designer: Doc update for 2.14.0 (except for release notes) (#137)
mulesoft/docs-design-center: modules/ROOT/pages/design-publish.adoc

<2> The middle pane, which describes the specification. You write the description in Exchange after publishing the specification.
<3> The right pane, which displays metadata about the specification.

== Before You Begin

> It is a good practice to remove unreferenced files before publishing a project to Anypoint Exchange. A unreferenced file is a file that meets both of these conditions:

Reviewer: AN unreferenced?

* It is not referenced from a project's root file.

> * It is not referenced from any file that the root file references.

Reviewer: re-thinking about this....maybe it is clearer to say "referenced directly or indirectly from the root file"...otherwise is sounds like you can reference it directly or from the level one referenced files, but not from a reference of a reference

+
<1> *Asset version*: The version number to be displayed in the *Version* section of the page that Anypoint Exchange creates for the project.
<2> *API version*: The version number of the API specification. For a description of this field, see the "Filling in the *API version* Field" section.

> <3> *Business group ID*: The ID of the business unit in Anypoint Platform to which the project belongs. You should keep the value as is, except in unusual cases.

Reviewer 2: "as is" is idiomatic but very common. I can envision my former English conversation students in Japan being stumped by it, but...your call.

<4> *Asset name*: The name that Anypoint Exchange uses to identify the published project. You should keep the value as is, except in unusual cases.
<5> *Asset ID*: The ID that Anypoint Exchange uses to identify the published project. You should keep the value as is, except in unusual cases.
<6> *Add to API Manager*: Specifies that you want to publish the API specification to an API portal. When you click *Publish to Exchange*, API Designer also publishes the specification to API Manager, where the specification is listed in the *API Administration* section.

> . After filling in required fields, click *Publish to Exchange*.

Reviewer 2: Also an internationalization issue:
```suggestion
. After completing the required fields, click *Publish to Exchange*.
```

> +

Reviewer 2: For similar reasons, I would omit "do in fact."
